src/libpgmoneta/walfile/rm_xact.c

```diff
@@ -372,6 +372,30 @@ pgmoneta_wal_parse_xl_xact_parsed_abort_v15(struct xl_xact_parsed_abort* wrapper
    memcpy(&wrapper->data.v15.origin_timestamp, ptr, sizeof(timestamp_tz));
 }
 
+void
+pgmoneta_wal_parse_commit_record_v14(uint8_t info, struct xl_xact_commit* xlrec, struct xl_xact_parsed_commit_v14* parsed)
+{
+   parse_commit_record_v14(info, xlrec, parsed);
+}
+
+void
+pgmoneta_wal_parse_commit_record_v15(uint8_t info, struct xl_xact_commit* xlrec, struct xl_xact_parsed_commit_v15* parsed)
+{
+   parse_commit_record_v15(info, xlrec, parsed);
+}
+
+void
+pgmoneta_wal_parse_abort_record_v14(uint8_t info, struct xl_xact_abort* xlrec, struct xl_xact_parsed_abort_v14* parsed)
```

Comment:
I know that v14 works for <14 as well, but the naming is a bit confusing. I know it's because it was named that way initially. But now that you have made the API public, try to make the naming clearer. That being said, I don't have good ideas rn..